backup.sh

```
@@ -1,4 +1,8 @@
 #!/bin/bash
+<< help
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)